src/vxingest/netcdf_to_cb/netcdf_builder.py

```diff
@@ -639,7 +639,37 @@ def umask_value_transform(self, params_dict):
             )
             return None
 
-    def handle_wind_dir_U(self, params_dict):
+    def handle_rh(self, params_dict):
+        """Retrieves a RH value, if there is one,
+           and if not derives the RH from the dewpoint and temperature.
+        Args:
+            params_dict (dict): named function parameters
+        Returns:
+            float: the RH
+        """
+        try:
+            # dewpoint = self.umask_value_transform({"recNum": params_dict["recNum"], "dewpoint":"dewpoint"})
+            # temperature = self.umask_value_transform({"recNum": params_dict["recNum"], "temperature":"temperature"})
+            dewpoint = self.umask_value_transform(
+                {"recNum": params_dict["recNum"], "dewpoint": params_dict["dewpoint"]}
+            )
+            temperature = self.umask_value_transform(
+                {
+                    "recNum": params_dict["recNum"],
+                    "temperature": params_dict["temperature"],
+                }
+            )
+            _q = (
+                relative_humidity_from_dewpoint(
+                    temperature * units.kelvin, dewpoint * units.kelvin
+                ).magnitude
+            ) * 100
+            return _q
+        except Exception as _e:  # pylint:disable=broad-except
+            # there must not have been one
+            return None
+
+    def handle_wind_dir_u(self, params_dict):
         """Retrieves a wind direction U value, if there is one,
            and if not derives the U component from the wind direction and speed.
            expects wind speed and wind direction to be in the params_dict.
```

Comment:
Similar comment for the `handle_wind_dir_u` and `handle_wind_dir_v` functions - I may be missing it but they don't appear to be retrieving `u` and `v`; they go directly to calculating them from the speed & direction instead.

Reply:
same as RH. I should have paid more attention to the docstring